### DISCUSSION TITLE
Retagging old image 3889723632

### DISCUSSION THREAD

**Discussion by CristopherVA** [dokku/dokku#6092] [General]
chown: changing ownership of '/app/src/components/MessageError.js': No space left on device
chown: changing ownership of '/app/src/components/PosTable/index.js': No space left on device
chown: changing ownership of '/app/src/components/PosTable': No space left on device
chown: changing ownership of '/app/src/components/BranchOfficeSelect/index.js': No space left on device
chown: changing ownership of '/app/src/components/BranchOfficeSelect': No space left on device
chown: changing ownership of '/app/src/components/ProductTypeSelect/index.js': No space left on device
chown: changing ownership of '/app/src/components/ProductTypeSelect': No space left on device
chown: changing ownership of '/app/src/components/PendingAppointments/index.js': No space left on device
chown: changing ownership of '/app/src/components/PendingAppointments': No space left on device
chown: changing ownership of '/app/src/components/CompanySelect/index.js': No space left on device
chown: changing ownership of '/app/src/components/CompanySelect': No space left on device
chown: changing ownership of '/app/src/components/IntermitentButton/index.js': No space left on device
chown: changing ownership of '/app/src/components/IntermitentButton': No space left on device
chown: changing ownership of '/app/src/components': No space left on device
chown: changing ownership of '/app/src/_nav.js': No space left on device
chown: changing ownership of '/app/scripts/deploy.sh': No space left on device
chown: changing ownership of '/app/scripts/run.sh': No space left on device
chown: changing ownership of '/app/scripts': No space left on device

**Comment by josegonzalez** [comment]
Do you... have more context as to what you're asking here?
